src/BaGetter.Core/Configuration/BaGetterOptions.cs

```diff
@@ -49,11 +49,7 @@ public class BaGetterOptions
     /// </summary>
     public uint MaxPackageSizeGiB { get; set; } = 8;
 
-    /// <summary>
-    /// If this is set to a value, it will limit the number of versions that can be pushed for a package.
-    /// the older versions will be deleted.
-    /// </summary>
-    public uint? MaxVersionsPerPackage { get; set; } = null;
```

Comment:
What about people, who are currently using this setting? You must set it deprecated/obsolete.
If they are using it currently you should fill the new `RetentionOptions` with the value of `MaxVersionsPerPackage`. A message in the console should point them to the problem so they can move to the new configuration.

Reply:
Agree - I was hoping this PR would be rolled in release before my other one that had the `MaxVersionsPerPackage`